assignment1/src/main/java/ro/utcn/sd/he/assignment1/persistence/jdbc/JdbcAnswerRepository.java

```diff
@@ -0,0 +1,79 @@
+package ro.utcn.sd.he.assignment1.persistence.jdbc;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import ro.utcn.sd.he.assignment1.model.Answer;
+import ro.utcn.sd.he.assignment1.model.Question;
+import ro.utcn.sd.he.assignment1.persistence.api.AnswerRepository;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+@RequiredArgsConstructor
+public class JdbcAnswerRepository implements AnswerRepository {
+    private final JdbcTemplate template;
+    private RowMapper<Answer> rowMapper = ((resultSet, i) -> new Answer(
```

Comment:
Nice! You are the first student to actually have this as a field. It could be `final` though, right? :)

Reply:
Well, you suggested it in the last feedback and I thought it was a very good idea. I also made it final.